style.css

```diff
@@ -0,0 +1,100 @@
+*{
+    margin:0;
+    padding: 0;
+    box-sizing:border-box; 
+}
+
+body{
+    background-color:indianred;
+}
+
+.name{
+    max-width: fit-content;
+    height: 2.5rem;
+    color: black;
+    font-weight: 200;
+    border: 5px dotted floralwhite;
+}
+
+.container{
+    align-items: center;
+}
```

Comment:
Add a space after this closing curly brace

Reply:
okay.. thanks